test/clangd_tests.vim

```diff
@@ -975,7 +975,7 @@ def g:Test_LspHover()
   popup_close(p[0])
   cursor(7, 1)
   output = execute(':LspHover')->split("\n")
-  assert_equal('No hover messages found for current position', output[0])
+  assert_match('No hover messages found for current position', output[0])
```

Comment:
Can you add a test for `LspHover!` as well?

    output = execute(':LspHover!')->split("\n")
    assert_equal('', output[0])

Reply:
as you said, it is async, actually i do not think that assert really worked.
// maybe we can refactor/remove such assert later.

Reply:
The tests are not async because g:LSP_Test is set to 1

Reply:
good catch.